src/stylesheets/components/_subnav.scss

```diff
@@ -16,44 +12,52 @@
   }
 
   .app-subnav__link {
-    display: block;
-    padding: 8px govuk-spacing(6) 8px govuk-spacing(2);
-    border-left: 4px solid transparent;
+    padding: 2px 0;
     text-decoration: none;
 
-    &:focus {
-      background: inherit;
-    }
-
-    &:hover,
-    &:active,
-    &:visited {
+    &:not(:focus):hover {
       color: $govuk-link-colour;
+      text-decoration: underline;
     }
+  }
 
-    &:hover {
-      border-left-color: $govuk-link-hover-colour;
-    }
+  .app-subnav__section-item {
+    margin-bottom: govuk-spacing(1);
+    padding-top: govuk-spacing(1);
+    padding-bottom: govuk-spacing(1);
+  }
+
+  .app-subnav__section-item--current {
+    margin-left: -(govuk-spacing(2) + 4px);
+    padding-left: govuk-spacing(2);
+    border-left: 4px solid govuk-colour("blue");
+    background-color: govuk-colour("white");
   }
 
   .app-subnav__section-item--current .app-subnav__link {
-    border-left-color: $govuk-link-colour;
-    background: $app-light-grey;
     font-weight: bold;
   }
 
   .app-subnav__section--nested {
+    margin-top: govuk-spacing(2);
     margin-bottom: 0;
+    padding-left: govuk-spacing(4);
+  }
+
+  .app-subnav__section--nested .app-subnav__section-item:before {
+    content: "—";
```

Comment:
The native focus in IE8 renders this a bit out of place 
<img width="150" alt="Screen Shot 2019-07-19 at 15 01 58" src="https://user-images.githubusercontent.com/5007934/61540841-615dbd80-aa36-11e9-8d0e-7045b4a1e373.png">

Reply:
Hmm, not sure we'll be able to resolve this :(

Reply:
What if we don't show the dash on IE8?

Reply:
I think given that the Design System website has a different audience to GOV.UK Frontend we should be OK with this being a little bit weird.

Reply:
The focused content is not clearly focused which makes it less accessible.  Wrapping the pseudo element in `govuk-not-ie8` would give a better experience:

<img width="337" alt="Screen Shot 2019-07-19 at 17 01 10" src="https://user-images.githubusercontent.com/5007934/61549066-0a141900-aa47-11e9-98a0-858fc8ae105b.png">

Reply:
I meant more, I don't think we have many (or any) users of IE8 that browse the website.

The website I think would fall under the service manual support matrix:
https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in

Happy to make this adjustment but wonder if we should have a clearer strategy of IE8 on the website since the layout is very broken currently.

Reply:
Deciding where we're going with IE8 with regards to this website sounds like a good idea 👍

Reply:
Resolved this by using the proper pseudo selector which doesnt work in IE8